src/ui/public/dialogues/safe_confirm.factory.js

```diff
@@ -0,0 +1,11 @@
+import uiModules from 'ui/modules';
+import { safeConfirm } from './safe_confirm';
+import './directives/confirm_dialogue';
+import './modal_dialogue.factory';
+
+const module = uiModules.get('kibana');
+
+module.factory('safeConfirm', function ($timeout, ModalDialogue, Promise) {
+  return (message, confirmButtonText = 'Okay', cancelButtonText = 'Cancel') =>
```

Comment:
Ahh, I understand your point better now and I'm sorry for reading your original comment too quickly - after re-reading it, you made yourself clear, so that was my bad.  I thought you wanted me to propagate 'Okay' and 'Cancel' everywhere.

I'm all for it, but since this PR does match prior functionality, I'd prefer to make the change in a separate PR.